### DISCUSSION TITLE
After deploying to AWS

### DISCUSSION THREAD

**Discussion by dongri727** [serverpod/serverpod#1751] [Q&A] [answered]
What can I do from here ? 
<img width="448" alt="スクリーンショット 2024-01-15 7 27 13" src="https://github.com/serverpod/serverpod/assets/84567162/b840d2ab-6d89-4e79-8e03-8e863b46ad86">

**Comment by binemmanuel** [comment, marked as answer]
Use "https://app.laporte.academy" as the base URL on your client-side app

**Reply by dongri727** [reply]
Thank You ! 　I'll move on !
